conode/conode.go

```diff
@@ -22,6 +22,7 @@ import (
 	_ "github.com/dedis/cothority/cosi/service"
 	_ "github.com/dedis/cothority/guard/service"
 	_ "github.com/dedis/cothority/identity"
+	_ "github.com/dedis/cothority/pop/service"
```

Comment:
we could even do a `cothority_hacks` or `cothority_samples` or such repo where we put that code inside. Then we can say that every change in ONet needs to be reflected in `cothority_template`, then `cothority` and finally `cothority_hacks`.